SerreFinitenessTheorem.agda

```diff
@@ -37,3 +40,19 @@ mutual
   saf→saf<-> X safX scX (suc n) =
     safTotal (<->EMFibSeq X n) (Conn→ConnConnCov X 3 (suc n) scX)
       (saf→saf<-> X safX scX n) (isFP→safEM (πAb n X) (saf→isFPπ X safX scX n) n)
+
+isFPπAbSn : (n m : ℕ) → isFP (πAb n (S (2 + m)))
```

Comment:
Is this the final statement that we want? I guess we could get for arbitrary m if we want by case-splitting and handle the two low dimensional cases by hand...